client/landing/stepper/declarative-flow/transferring-hosted-site-flow.ts

```diff
@@ -1,5 +1,7 @@
 import { TRANSFERRING_HOSTED_SITE_FLOW } from '@automattic/onboarding';
 import { useDispatch } from '@wordpress/data';
+import { useDispatch as dispatcher } from 'react-redux';
```

Comment:
Feels weird doing this, can we achieve same with `useDispatch` from `@wordpress/data`

Reply:
It seems like this is the correct way. I did make a small change and called the alias `useReduxDispatch`.